src/or_trajopt/planning.py

```diff
@@ -58,7 +59,8 @@ class CostType(enum.Enum):
 
 
 class TrajoptWrapper(MetaPlanner):
-    def __init__(self, planner):
+    def __init__(self, planner,
+            robot_collision_checker=DefaultRobotCollisionChecker):
```

Comment:
What do you want the indentation to look like? I am fine with changing it, but it looks correct to me. 😕